JSDemos/Demos/TreeList/BatchEditing/description.md

```diff
@@ -1 +1,10 @@
-Batch editing is&nbsp;very similar to&nbsp;cell editing in&nbsp;that a&nbsp;user edits data cell by&nbsp;cell. The only difference is&nbsp;that with batch editing a&nbsp;user needs to&nbsp;click the _&laquo;Save changes&raquo;_ button to&nbsp;actually save changes. Only then the TreeList will collect all changes in&nbsp;a&nbsp;batch and send them to&nbsp;the data source.
\ No newline at end of file
+You can use the batch edit mode to defer saving multiple changes. In this mode, users edit data one cell at a time. The changes are stored in a buffer and can be discarded before the **Save** button is clicked.
+
+To enable the batch edit mode, configure the following properties:
+
+* Set **editing**.[mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#mode) to *"batch"*.
+* Assign **true**  to the [editing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#mode) object's [allowUpdating](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#allowUpdating), [allowAdding](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#allowAdding), and [allowDeleting](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#allowDeleting) properties.
+
+This demo uses the [onInitNewRow](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#onInitNewRow) handler to populate cells of a new row with default values. This function sets *"Not Started"* as the initial task status and specifies the current date as the initial start and due dates.
```

Comment:
We already described the same functionality for the [Cell Editing](https://github.com/DevExpress/devextreme-demos/pull/739/files#diff-a51aa346a8fb80eb61c2d3f47d92b3e43c28ab11c81ea46c17c44b09036ed4b4R7) demo.

```suggestion
This demo also shows how to populate cells of a new row with default values. The values are assigned within the [onInitNewRow](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#onInitNewRow) event handler.
```